starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,66 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
-
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
-
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
+<div>
+<div class= "apple">
+<img src="./images/apple-pie.jpg" alt="apple pie">
+<h1>Apple Pie</h1>
+</div>
+
+<div class="apple-p">
+<p>This was my grandmother's apple pie recipe. I have never seen another one
+  quite like it. It will always be my favorite and has won me several first place
+  prizes in local competitions. I hope it becomes one of your favorites as well!
+  </p>
+</div>
+
+<div class="graph">
+  <img src="./images/recipe-info.png" alt="recipe info">
+</div>
+
+<hr>
+
+<br>
+
+<div class= "ingredients">
+<h3>Ingredients</h3>
+<ul>
+  <li>1 recipe pastry for a 9 inch double crust pie</li>
+  <li>1/2 cup unsalted butter</li>
+  <li>3 tablespoons all-purpose flour</li>
+  <li>1/4 cup water</li>
+  <li>3 tablespoons all-purpose flour</li>
+  <li>1/4 cup water</li>
+  <li>1/2 cup white sugar</li>
+  <li>1/2 cup packed brown sugar</li>
+  <li>8 Granny Smith apples - peeled, cored and sliced</li>
+</ul>
+</div>
+
+<div class="info">
+    <img src="./images/cooking-info.png" alt="cooking info">
+</div>
+
+<hr>
+
+<br>
+
+<h3>Directions</h3>
+<ol>
+<li>Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
```

Comment:
list above these should have started with the number and ) instead of .(period), a quick work around is to put `list-style: none` and manually enter the 1) 2) etc. since its a short list.